### ISSUE TITLE
Flask startup error using debug=True and Werkzeug version >= 0.15.0

### ISSUE THREAD

**Issue by cleberangelo** [pallets/werkzeug#1504]
There's an error occurring when startup an application using Flask (with create_app as the application factory function) and Werkzeug version >=  **0.15.0**. Downgrade to version **0.14.1** solves that. If debug=False, no error occurs.

```
* Serving Flask app "almox" (lazy loading)
 * Environment: dev
 * Debug mode: on
 * Running on http://127.0.0.1:8082/ (Press CTRL+C to quit)
 * Restarting with stat
Traceback (most recent call last):
  File "run.py", line 6, in <module>
    app.run(port=8082)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/flask/app.py", line 943, in run
    run_simple(host, port, self, **options)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/werkzeug/serving.py", line 988, in run_simple
    run_with_reloader(inner, extra_files, reloader_interval, reloader_type)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/werkzeug/_reloader.py", line 332, in run_with_reloader
    sys.exit(reloader.restart_with_reloader())
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/werkzeug/_reloader.py", line 176, in restart_with_reloader
    exit_code = subprocess.call(args, env=new_environ, close_fds=False)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 323, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 1522, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
OSError: [Errno 8] Exec format error: '/Users/cleber/Devel/Python/Flask/flask-almox/run.py
```

**Comment by davidism** [comment]
duplicate of #1482 

Don't mark your script executable if it doesn't have a `#!/usr/bin/env python` line at the top, and prefer using the `flask run` command.

**Comment by aaveidt** [comment]
@davidism `#!/usr/bin/env python` doesn't work on Mac.

**Comment by davidism** [comment]
Fixed in Werkzeug 0.15.5.
